tango_simlib/tests/weather_supplementary_SIMDD.json

```diff
@@ -63,7 +63,7 @@
     },
     {
       "basicAttributeData": {
-        "name": "relative_humidity",
+        "name": "relative-humidity",
         "dataSimulationParameters": {
```

Comment:
Try to get a naming convention for these interface description files. I suggest Weather.xmi and Weather_SIMDD.json. This aligns with Dish.xmi and Dish_SIMDD.json used for DishElement.
Or is there anything in weather_sim.xmi that is not the standard weather TANGO interface and is simulator specific? If so, then may be Weather_sim.xmi and Weather_SIMDD.json.